front-end/src/components/svg-icon/SvgIcon.js

```diff
@@ -0,0 +1,120 @@
+            <g id="Group" stroke="#979797">
+                <rect id="Rectangle" x="0.5" y="0.5" width="18" height="19" rx="2"></rect>
+                <rect id="Rectangle" fill="#9B9B9B" x="3.66666667" y="3.65789474" width="4.27777778" height="4.26315789" rx="1"></rect>
+                <path d="M10.0277778,3.68421053 L15.3055556,3.68421053" id="Line" strokeLinecap="square"></path>
+                <path d="M10.0277778,5.51315789 L15.3055556,5.51315789" id="Line" strokeLinecap="square"></path>
+                <path d="M10.0277778,7.61842105 L15.3055556,7.61842105" id="Line" strokeLinecap="square"></path>
+                <path d="M3.69444444,9.73684211 L15.3055556,9.73684211" id="Line" strokeLinecap="square"></path>
+                <path d="M3.69444444,11.7872807 L15.3055556,11.7872807" id="Line" strokeLinecap="square"></path>
+                <path d="M3.69444444,13.997807 L15.3055556,13.997807" id="Line" strokeLinecap="square"></path>
+                <path d="M3.69444444,16.2083333 L15.3055556,16.2083333" id="Line" strokeLinecap="square"></path>
+            </g>
+        </g>
+    </svg>
+);
+
+const newsIconActive = (
+    <svg width="19px" height="20px" viewBox="0 0 19 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink">
+        {/* Generator: Sketch 51.3 (57544) - http://www.bohemiancoding.com/sketch */}
+        <title>Glyphs / Bar Button / news icon/ selected</title>
+        <desc>Created with Sketch.</desc>
+        <defs></defs>
+        <g id="Glyphs-/-Bar-Button-/-news-icon/-selected" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+            <rect id="Rectangle" stroke="#3C8FDE" x="0.5" y="0.5" width="18" height="19" rx="2"></rect>
+            <rect id="Rectangle" stroke="#3C8FDE" fill="#3E92CF" x="3.66666667" y="3.65789474" width="4.27777778" height="4.26315789" rx="1"></rect>
+            <path d="M10.0277778,3.68421053 L15.3055556,3.68421053" id="Line" stroke="#3C8FDE" strokeLinecap="square"></path>
+            <path d="M10.0277778,5.51315789 L15.3055556,5.51315789" id="Line" stroke="#3C8FDE" strokeLinecap="square"></path>
+            <path d="M10.0277778,7.61842105 L15.3055556,7.61842105" id="Line" stroke="#3C8FDE" strokeLinecap="square"></path>
+            <path d="M3.69444444,9.73684211 L15.3055556,9.73684211" id="Line" stroke="#3C8FDE" strokeLinecap="square"></path>
+            <path d="M3.69444444,11.7872807 L15.3055556,11.7872807" id="Line" stroke="#3C8FDE" strokeLinecap="square"></path>
+            <path d="M3.69444444,13.997807 L15.3055556,13.997807" id="Line" stroke="#3C8FDE" strokeLinecap="square"></path>
+            <path d="M3.69444444,16.2083333 L15.3055556,16.2083333" id="Line" stroke="#3C8FDE" strokeLinecap="square"></path>
+        </g>
+    </svg>
+);
+
+const getSvg = (name, isActive) => {
+    switch(name) {
+        case 'newsIcon': 
+            return isActive ? newsIconActive : newsIcon;
```

Comment:
if it is not too much change between the two state, you can just use css!

Reply:
Css was tried, but requires some ugly hacks to properly style the svg elements, particularly for the news feed icon. Found it easier to just go this way.

Reply:
Maybe i can help xD